### PR TITLE
Improve drawing screen

### PR DESCRIPTION
**Pull request by vkallerud**
Updated drawing screen according to apprived figmadesign.
Added components for the I and O (these are not finished, will have to do after demo).

### CODE REVIEW

File: src/assets/icons/icons8-trash-144.png
Comment (andreasrberg) [inline src/assets/icons/icons8-trash-144.png:1]: We should probably look at an icon library to use across the project eventually

File: src/assets/avatars/o-avatar/o-avatar.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-o-avatar',
+  templateUrl: './o-avatar.component.html',
+  styleUrls: ['./o-avatar.component.scss'],
+  standalone: true,
+})
+export class OAvatarComponent {
+  @Input() color: CustomColorsIO = CustomColorsIO.black; // Default color
+
+  height = 192;
+  width = 192;
```
Comment (andreasrberg) [inline src/assets/avatars/o-avatar/o-avatar.component.ts:14]: Remember to add this to the HTML aswell. Now it is defined both here and statically in the o-avatar html

File: src/app/game/game-draw/game-draw.component.ts [hunk 2]
```diff
@@ -14,20 +21,33 @@ import { GameConfigService } from '../services/game-config.service';
...
+  ],
 })
 export class GameDrawComponent implements OnInit, OnDestroy {
   config!: GameLevelConfig;
   subscriptions = new Subscription();
 
-  canvas = viewChild.required<ElementRef<HTMLCanvasElement>>('canvas');
-  countDown = viewChild.required<ElementRef<HTMLSpanElement>>('countDown');
+  @ViewChild('canvas', { static: true }) canvas!: ElementRef<HTMLCanvasElement>;
+  @ViewChild('countDown', { static: true }) countDown!: ElementRef<HTMLSpanElement>;
```
Comment (andreasrberg) [inline src/app/game/game-draw/game-draw.component.ts:50]: Can this eventually be done with question marks instead? Me no like exclamation marks 🥴

**Review by andreasrberg — APPROVED** [review]
A few small comments to look at, but looks great!

File: src/assets/avatars/o-avatar/o-avatar.component.ts (hunk 1 above)
Comment (vkallerud) [reply src/assets/avatars/o-avatar/o-avatar.component.ts:14]: Yeah i know. Just pushed an unfinished solution, but will complete them later hehe

**Review by sondrefj — APPROVED** [review]

**Pull request closed without merging**